app/controllers/uploads_controller.rb

```diff
@@ -16,8 +16,12 @@ def create
     # capture current user for block later on
     me = current_user
 
+    params.permit(:type, :upload_type)
+    if params[:type].blank? && params[:upload_type].blank?
```

Comment:
curious, why do we need a different param here?

Reply:
Can you take a look at the comments I left above please? I addressed this there.